docs/Troubleshooting.md

```diff
@@ -378,3 +381,27 @@ After downloading the license file perform the following steps:
 Information about licenses from the [SAS Viya Platform Operations Guide](https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=default&docsetId=k8sag&docsetTarget=n14rkqa3cycmd0n1ub50k47x7lbb.htm)
 
 Note, these steps are only applicable for updating your license file, if you are going to be updating the SAS deployment or including additional products in your order we recommend that your perform your update manually. See this note in the [README](https://github.com/sassoftware/viya4-deployment#updating-sas-viya-manually)
+
+
+## PostgreSQL Data Transfer Guide - Post Migration Steps
+
+### Symptom:
+
+You had an existing SAS Viya platform deployment that was created using the viya4-deployment project and configured to use Crunchy as the database, and you have now completed the manual steps outlined in the "PostgreSQL Data Transfer Guide" to move your data over to an external PostgreSQL cluster. You now want to use the viya4-deployment project to manage your installation again.
+
+### Solution:
+
+1. After following the "PostgreSQL Data Transfer Guide" under "Steps for Crunchy Data 5" you manually added entries for the external PostgreSQL clusters into the kustomization.yaml and produced a file that includes details for **both** the internal Crunchy and external Postgres instance. You then built and applied the manifest so that the Viya deployment is reconfigured to point to the external Postgres instance.
+2. The final step the "PostgreSQL Data Transfer Guide" under "Steps for Crunchy Data 5"  asks you manually remove entries to related to Crunchy and rebuild your manifests again. At this point in time you could instead remove the manual modifications you made to the kustomization.yaml and make changes to your `ansible-vars.yaml` so that the viya4-deployment project can manage your installation again.
```

Comment:
just a minor wording suggestion
```suggestion
2. The final step in the "PostgreSQL Data Transfer Guide" under "Steps for Crunchy Data 5"  asks you to manually remove entries to related to Crunchy and rebuild your manifests again. At this point in time you could instead remove the manual modifications you made to the kustomization.yaml and make changes to your `ansible-vars.yaml` so that the viya4-deployment project can manage your installation again.
```

Reply:
Thanks for the fix, committed the changes.